plugins/split-yaml.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: split-yaml
+spec:
+  version: v0.0.1
+  homepage: https://github.com/nathforge/kubectl-split-yaml
+  shortDescription: Save Kubernetes YAML output into one file per resource.
+  description: This plugin takes output from `kubectl get -o yaml` and saves one file
+    per resource.
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/nathforge/kubectl-split-yaml/releases/download/v0.0.1/kubectl-split-yaml_0.0.1_darwin_amd64.tar.gz
+    sha256: 05ad495bb6045355c064235e945f9b1860f181046f1cc1d7fc75199bafb820c5
+    files:
+    - from: '*'
+      to: .
```

Comment:
this statement is redundant since that's the default, you can remove.

Reply:
Thanks, have done.